lib/tizen_plugins.dart

```diff
@@ -265,6 +265,7 @@ Future<void> ensureReadyForTizenTooling(FlutterProject project) async {
   await tizenProject.ensureReadyForPlatformSpecificTooling();
 
   await injectTizenPlugins(project);
+  await _noticeAvailableTizenPlugins(project);
```

Comment:
"notice" means "become aware of".
```suggestion
  await _informAvailableTizenPlugins(project);
```